docs/config/extras.md

```diff
@@ -104,6 +104,7 @@ with the intent of making these the default behavior in Stencil v5. When set to
 - Failed to execute 'removeChild' on 'Node' [(#3278)](https://github.com/ionic-team/stencil/issues/3278) (since v4.9.0)
 - React fails to manage children in Stencil slot [(#2259)](https://github.com/ionic-team/stencil/issues/2259) (since v4.9.0)
 - Slot name is not updated when it is bind to a prop [(#2982)](https://github.com/ionic-team/stencil/issues/2982) (since 4.12.1)
+- Conditionally rendered slots not working [(#5335)](https://github.com/ionic-team/stencil/issues/5335) (since 4.13.0)
```

Comment:
The PR itself looks good to me, so I'm making assumptions about the version here